src/zone.ts

```diff
@@ -0,0 +1,80 @@
+import { parseZoneFile, makeZoneFile } from "zone-file";
+import * as fs from 'fs';
+
+export function getDefaultAddress(coin: string, text: any | string): { [key: string]: string } | null {
+    let addresses = getAddresses(coin, text);
+    return (addresses == null ? null : addresses[0]);
+}
+
+export function getAddresses(coin: string, text: any | string): { [key: string]: string }[] | null {
```

Comment:
For the return type we can probably just use `Array<string>`

Reply:
ya that also works - the original standard returned a dictionary but array is clean if we only pass in a single coin as argument